src/text/Text.js

```diff
@@ -25,21 +25,15 @@ const TextElement = ({style, children, h1, h2, h3, h4, h5, h6, fontFamily}) => (
 styles = StyleSheet.create({
   text: {
     ...Platform.select({
-      ios: {
-        fontFamily: fonts.ios.regular
-      },
       android: {
-        fontFamily: fonts.android.regular
+        ...fonts.android.regular
       }
     })
   },
   bold: {
     ...Platform.select({
-      ios: {
-        fontFamily: fonts.ios.bold
-      },
       android: {
-        fontFamily: fonts.android.bold
+        ...fonts.android.bold
```

Comment:
Its more like a doubt than a comment. How does this work? I mean how does the spread operator replace `fontFamily`?
```
-        fontFamily: fonts.android.bold
+        ...fonts.android.bold
```

I've used spread operator in different places, was curious to know how this worked.